content/ja/handbook/engineering/architecture/design-documents/cells/goals.md

```
@@ -0,0 +1,425 @@
+今日のカナリアデプロイの主な利点は、内部ユーザーに先にコード変更をロールアウトできることです。
+Cells では、内部ユーザーが特定の Cell に存在し、その Cell を最初にデプロイできます。
+
+さらに Cells は今日のカナリアにおける一部の問題を解決するかもしれません。
+例えば、ユーザーのサブセットに対して Sidekiq コードをアップグレードする方法がないといった問題です。
+
+## 用語集
+
+| 用語 | 説明 | 非推奨用語 |
+| ----------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------ |
+| [Cell](#cell) | Cell とは、異なる Organizations に属する複数のトップレベルグループを含むインフラストラクチャコンポーネントのセットです。 | GitLab インスタンス、クラスター、シャード、Pod |
+| [レガシー Cell](#legacy-cell) | 既存の GitLab.com インフラストラクチャ | プライマリ Cell |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)